packages/p2p-v2/src/pages/buy-sell/screens/BuySellTable/BuySellTable.tsx

```diff
@@ -23,13 +25,28 @@ const headerRenderer = (header: string) => <span>{header}</span>;
 
 const BuySellTable = () => {
     const [activeTab, setActiveTab] = useState<string>('Buy');
+    const [sortDropdownValue, setSortDropdownValue] = useState<TSortByValues>('rate');
+    const [isFilterModalOpen, setIsFilterModalOpen] = useState<boolean>(false);
     const { data, isFetching, isLoading, loadMoreAdverts } = p2p.advert.useGetList({
         counterparty_type: activeTab === 'Buy' ? BUY_SELL.BUY : BUY_SELL.SELL,
+        local_currency: 'BWP',
```

Comment:
local_currency value?

Reply:
ah forgot to remove this, i was checking the BWP currency